src/test/java/net/oauth/jsontoken/JsonTokenParserTest.java

```diff
@@ -0,0 +1,4 @@
+package net.oauth.jsontoken;
+
+public class JsonTokenParserTest {
+}
```

Comment:
Revert this file for this PR?

Reply:
Yeah, didn't mean to add this in the commit. Removed